src/drivers/imx/interrupt.c

```diff
@@ -363,6 +363,18 @@ static const struct irq_cascade_tmpl dsp_irq[] = {
 	IRQSTR_CASCADE_TMPL_DECL(7)
 };
```

Comment:
title should say: drivers: imx: interrupt: Convert IRQ steer interrupt to logical SOF interrupt